aion/web/router.ex

```diff
@@ -42,6 +42,7 @@ defmodule Aion.Router do
     pipe_through [:api, :api_auth]
 
     get "/me", UserController, :get_user_info
+    get "/ranking", RankingController, :ranking
```

Comment:
If you used `Ecto.Schema` it would also transfer to `resources` (where you can easily control which parts of CRUD we want to enable.